ui/media/js/engine.js

```diff
@@ -106,8 +110,10 @@
                     const readState = await reader.read()
                     value = this.parse(readState.value)
                     if (value) {
-                        for(let sVal of value) {
-                            ({value: sVal, done} = await Promise.resolve(this.onNext({value:sVal, done:readState.done})))
+                        for (let sVal of value) {
+                            ;({ value: sVal, done } = await Promise.resolve(
```

Comment:
Do you know why it adds a semicolon to the start of the line? It did that at the start of the engine.js as well.